docs/releases.rst

```diff
@@ -0,0 +1,20 @@
+********************************************************************************
+Releases
+********************************************************************************
+
+.. rst-class:: lead
+
+COMPAS releases are available on the Python Package Index (PyPI), on ``conda-forge``
+or directly from the Github repo.
+
+Versions are tagged following the Semantic Versioning (semver) convention of ``MAJOR.MINOR.PATCH``.
+For more information see `Semantic Versioning 2.0.0 <https://semver.org/>`_.
+
+The current version of COMPAS is ``0.19.1``, which means we have not yet established a compleltely stable API.
+However, the ``0.19`` version series is the last before the release of ``COMPAS 1.0.0``!
+
+In the future, in addition to a version number, major releases will also have a name following the pattern
+``<geometric adjective> <nautical term>``, with the words forming an alliteration and the alliteration letter correspoding to the number of the release.
```

Comment:
```suggestion
``<geometric adjective> <nautical term>``, with the words forming an alliteration and the alliteration letter corresponding to the number of the release.
```
Also, I just learned this, but technically (pedantically) we are asking for some thing that is not only an alliteration but also a tautogram. https://en.wikipedia.org/wiki/Tautogram